plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -164,7 +164,7 @@
     private final Optional<String> defaultDir;
     private final String catalogId;
     private final int partitionSegments;
-    private final Executor partitionsReadExecutor;
+    private final Executor executor;
```

Comment:
are these renames important?

Reply:
I think `@ForGlueHiveMetastore` should take care about injection. no need for renaming @hashhar right?

Reply:
Yes. Sorry about the earlier confusion.